python/ray/worker.py

```diff
@@ -1854,14 +1854,14 @@ def method(self):
             infinite retries.
         runtime_env (Dict[str, Any]): Specifies the runtime environment for
             this actor or task and its children. See``runtime_env.py`` for
-            detailed documentation.
+            detailed documentation.  Note: can only be set via `.options()`.
```

Comment:
Make an issue to follow up and support this in the dictionary?

Reply:
Here's the issue: https://github.com/ray-project/ray/issues/15374